docs/classic-ui/theming/settings-ttw.md

```diff
@@ -0,0 +1,62 @@
+---
+myst:
+  html_meta:
+    "description": "Change Classic UI theme settings through-the-web (TTW) in Plone 6"
+    "property=og:description": "Change Classic UI theme settings through-the-web (TTW) in Plone 6"
+    "property=og:title": "Change Classic UI theme settings through-the-web (TTW) in Plone 6"
+    "keywords": "Plone 6, Classic UI, Through-the-web, TTW, theme, customization"
+---
+
+(classic-ui-through-the-web-label)=
+
+# Change theme settings TTW
+
+This chapter describes how to change Classic UI theme settings through-the-web ({term}`TTW`) in Plone 6.
+
+Small theme changes can be made via control panels or by updating Plone 6 Classic UI's `custom.css`.
```

Comment:
```suggestion
Small theme changes can be made via control panels.
```

There is no `custom.css` which you could update.

Reply:
There is a `custom.css` generated when you go through the process under [Custom Styles](https://plone6--1831.org.readthedocs.build/classic-ui/theming/settings-ttw.html#custom-styles). But, yeah, I see your point. It's an implementation detail that most users won't even know, unless they view the rendered HTML. Plus a Site Admin does it through a control panel anyway.

Reply:
Ah... I know what you mean now. It would be event more problematic if a user looks at the source code and tries to find the `custom.css` file anywhere, but doesn't physically exists because its generated. This should perhaps be explained in the chapter "custom styles"